domains/sudev.json

```diff
@@ -1,11 +1,10 @@
 {
-  "description": "Microsoft Student Ambassador, student at College of Engineering Trivandrum",
-  "repo": "https://github.com/gamegods3/gamegods3.github.io",
+  "repo": "https://github.com/mqry/mqry.github.io",
   "owner": {
-    "username": "gamegods3",
-    "email": "sudevssuresh@gmail.com"
+    "username": "mqry",
```

Comment:
You can't just take someones subdomain. I'll need @GameGodS3 to approve this if he doesn't want his domain.

Reply:
hey! sorry I just noticed that I did that as a mistake but I've fixed it hours ago by using my own json .. " mqry.json ", which has all the information.

Reply:
@mqry So you want me to revert the changes on the other file?

Reply:
yes, I meant to put all of those information onto my own file | "mqry.json"